COMMERCE-10598 Sanitize addresses for safe HTML viewing

elocamp opened the pull request:
The Address name and other fields were not being sanitized for HTML viewing.
Proposed fix is to sanitize the values and labels.

Related issues: [COMMERCE-10598](https://issues.liferay.com/browse/COMMERCE-10598)
cc: @victorcdp

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan1 commented:
ci:test:relevant

matthewchan1 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/4836/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4836/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 1 minute 3 seconds 17 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/eb9bbdd9ef9860b1e83b98123cd7d9b4ceeeccc6">eb9bbdd9ef9860b1e83b98123cd7d9b4ceeeccc6</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/4836/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4836/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/4836/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>The sender branch SHA could not be found on <strong><a href="https://github.com/matthewchan1/liferay-portal/tree/LPS-99662">matthewchan1/LPS-99662</a></strong>. The sender branch may have been force pushed or deleted after the pull request test was initiated.</p><pre><code>fatal: Not a valid branch point: 'f61c3c83729d8196b2e32bb1ba5465dab9b68657'.</code></pre></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

matthewchan1 commented:
ci:test:relevant

matthewchan1 commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-4-1.liferay.com/job/test-portal-source-format/323/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-4-1.liferay.com//userContent/jobs/test-portal-source-format/builds/323/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-1.liferay.com/job/test-portal-source-format/1032/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-1.liferay.com//userContent/jobs/test-portal-source-format/builds/1032/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes 28 seconds 677 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b3521ca44c8365d56c785c51c46b4f67a49a3581">b3521ca44c8365d56c785c51c46b4f67a49a3581</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/matthewchan1/liferay-portal/tree/LPS-99662">LPS-99662</a><br/>Branch GIT ID: <a href="https://github.com/matthewchan1/liferay-portal/commit/dd3b3f544cc4f14e96a7960662375134f510e460">dd3b3f544cc4f14e96a7960662375134f510e460</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-4-1.liferay.com/job/test-portal-source-format/323/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-4-1.liferay.com/userContent/jobs/test-portal-source-format/builds/323/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes 49 seconds 814 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b3521ca44c8365d56c785c51c46b4f67a49a3581">b3521ca44c8365d56c785c51c46b4f67a49a3581</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/matthewchan1/liferay-portal/tree/LPS-99662">LPS-99662</a><br/>Branch GIT ID: <a href="https://github.com/matthewchan1/liferay-portal/commit/dd3b3f544cc4f14e96a7960662375134f510e460">dd3b3f544cc4f14e96a7960662375134f510e460</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-1.liferay.com/job/test-portal-source-format/1032/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-1.liferay.com/userContent/jobs/test-portal-source-format/builds/1032/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest(master)/4997/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-19.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4997/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/4768/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-14.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4768/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 23 out of 25 jobs passed in 2 hours 49 seconds 244 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/eb9bbdd9ef9860b1e83b98123cd7d9b4ceeeccc6">eb9bbdd9ef9860b1e83b98123cd7d9b4ceeeccc6</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/c8fb62e2c71fe420e9c9f32f6e26d8f1ad741327">c8fb62e2c71fe420e9c9f32f6e26d8f1ad741327</a></p>23 out of 25 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest(master)/4997/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261633/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</strong></strike></a></li></ul></div><details><summary><strong>23 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308560/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261637/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/273180/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261638/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308559/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308558/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261636/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261635/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/273177/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261634/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323419/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328751/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/273181/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328750/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261632/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261639/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/316489/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/273179/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261631/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/273176/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308561/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328752/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/273178/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-19.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4997/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261633/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/261633//consoleText">AXIS_VARIABLE=0,label_exp=!master #261633</a><pre><code>     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:dynamic-data-mapping:dynamic-data-mapping-form-web:npmRunTest'.
     [exec] &gt; java.io.IOException: Process '[/opt/dev/projects/github/liferay-portal/build/node/bin/node, /opt/dev/projects/github/liferay-portal/modules/yarn-1.13.0.js, --production, false, --registry, http://mirrors.lax.liferay.com:4873, run, test]' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:dynamic-data-mapping:dynamic-data-mapping-form-web:npmRunTest'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [exec] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [exec] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)</code></pre></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 23 out of 25 jobs passed in 2 hours 4 minutes 45 seconds 516 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/eb9bbdd9ef9860b1e83b98123cd7d9b4ceeeccc6">eb9bbdd9ef9860b1e83b98123cd7d9b4ceeeccc6</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/c8fb62e2c71fe420e9c9f32f6e26d8f1ad741327">c8fb62e2c71fe420e9c9f32f6e26d8f1ad741327</a></p>23 out of 25 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/4768/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328755/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</strong></strike></a></li></ul></div><details><summary><strong>23 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328763/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328762/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328764/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/71049/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328761/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/71047/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/71046/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328759/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328758/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328757/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/71051/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328765/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/71050/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/184209/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328754/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286277/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/184211/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/71048/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328753/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328756/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/184210/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286278/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328760/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4768/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/328755/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/328755//consoleText">AXIS_VARIABLE=0,label_exp=!master #328755</a><pre><code>     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:dynamic-data-mapping:dynamic-data-mapping-form-web:npmRunTest'.
     [exec] &gt; java.io.IOException: Process '[/opt/dev/projects/github/liferay-portal/build/node/bin/node, /opt/dev/projects/github/liferay-portal/modules/yarn-1.13.0.js, --production, false, --registry, http://mirrors.lax.liferay.com:4873, run, test]' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:dynamic-data-mapping:dynamic-data-mapping-form-web:npmRunTest'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [exec] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [exec] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)</code></pre></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
Hey @matthewchan1 ,

Could you update the PR with more details? I will redirect the PR once that is completed. Thanks.

markgulacsy committed:
LPS-139647 Create new LayoutPageTemplateStructure upgrade process

ealonso committed:
LPS-149549 Removes no necessary condition

ealonso committed:
LPS-149549 Removes it, since the default value is asset list

Commit:
LPS-155688 Apply our new patch jar(hibernate.core-3.6.10.LIFERAY-PATCHED-7).

dacousalr committed:
LPS-139647 SF

ealonso committed:
LPS-139647 Inline

ealonso committed:
LPS-139647 Don't use implementation

ealonso committed:
LPS-139647 Simplify

ealonso committed:
LPS-139647 Inline

ealonso committed:
LPS-139647 Include select into the query instead of creating a dynamic query, a stream to get the plids and logic for batches because of Oracle

ealonso committed:
LPS-139647 Rename

ealonso committed:
LPS-139647 SF

ealonso committed:
LPS-139647 empty line

ealonso committed:
LPS-139647 Moves queries inside the same try block

ealonso committed:
LPS-139647 Rename

ealonso committed:
LPS-139647 Uses service to fetch layout and discard non draft layouts

brianchandotcom committed:
LPS-139647 SF

brianchandotcom committed:
LPS-139647 SF

ling-alan-huang committed:
LPS-139647 Auto SF

victorg1991 committed:
LPS-164384 Show display preview when viewing the content in global or in an asset library

# Conflicts:
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/servlet/taglib/ui/JournalDisplayPagePreviewFormNavigatorEntry.java

victorg1991 committed:
LPS-164384 Add the recent sites, fill the list until we have 6 sites

# Conflicts:
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java

victorg1991 committed:
LPS-164384 Create component to show the site selector in the display page preview

# Conflicts:
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java

victorg1991 committed:
LPS-164384 Add display page selector

# Conflicts:
#	modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/display/context/AssetDisplayPagesItemSelectorViewDisplayContext.java
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java

victorg1991 committed:
LPS-164384 AutoSF

# Conflicts:
#	portal-impl/src/portal.properties

victorg1991 committed:
LPS-164384 Return the plid of the display page

# Conflicts:
#	modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/servlet/taglib/clay/LayoutPageTemplateEntryVerticalCard.java

victorg1991 committed:
LPS-164384 Add preview button

# Conflicts:
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java

victorg1991 committed:
LPS-164384 Add preview tooltip

victorg1991 committed:
LPS-164384 Show default checkbox when selecting the display page

ealonso committed:
LPS-164384 build.gradle

victorg1991 committed:
LPS-164384 Hide Global site since it can't contain display pages

victorg1991 committed:
LPS-164384 Fix more button

victorg1991 committed:
LPS-164384 Show error when trying to open the preview

# Conflicts:
#	modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

victorg1991 committed:
LPS-164384 Rename

# Conflicts:
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/frontend/taglib/form/navigator/JournalDisplayPageFormNavigatorEntry.java

victorg1991 committed:
LPS-164384 Reset formDate to avoid errors if the user uses the "save draft" button

ealonso committed:
LPS-164384 We should filter by permissions, so don't call local service method

ealonso committed:
LPS-164384 It should be final

brianchandotcom committed:
LPS-164384 SF

victorg1991 committed:
LPS-165501 Don't show preview in edit default values

victorg1991 committed:
LPS-165472 Since we are removing the Global site from the list, this is always going to be minus one

victorg1991 committed:
LPS-165916 Return also the articleId of the newly created journal article

victorg1991 committed:
LPS-165916 Update articleId when creating the draft, take into account the article id to change the cmd

ces-quanhuynh committed:
LPS-164371 Fix lockout issue

victorg1991 committed:
LPS-166177 Fix newArticle condition

When we have default values for the structure the article is not null, but the articleId is

victorg1991 committed:
LPS-166177 Send also the friendlyURL

victorg1991 committed:
LPS-166451 Remove FF

victorg1991 committed:
LPS-166451 Auto SF

# Conflicts:
#	portal-impl/src/portal.properties

victorg1991 committed:
LPS-166451 Prevent null

In the global site the friendlyUrl input is not available

antonio91dev committed:
LPS-165197 - show exceptions for uppercase , symbols and numbers

antonio91dev committed:
LPS-163496 Set nextItemIndex as class variable so the value from the banner-slider active item won't be lost

vendeltoreki committed:
LPS-167777 Set the correct portletId for each item in PortletDataContext while publishing through ChangeSets (when "From Last Publish Date" option is selected in the form)

vendeltoreki committed:
LPS-167777 Add the "referenced content behavior" option to the publish/export form of DL files

balazssk committed:
LPS-164384 Fix package

balazssk committed:
LPS-164384 Adapt pre-LPS-161061 pattern

adolfopa committed:
LPS-166850 Check permissions before asking group count

It is possible a user that can edit web content doesn't have the
global VIEW permission over groups (e.g. only has VIEW permissions in
the current group). In this case, the `getGroupsCount` will fail with
a principal exception.

balazssk committed:
LPS-164384 Get Item selector - Partial of "LPS-160491 Use new item selector"

adolfopa committed:
LPS-166850 Avoid returning negative values

Although the user should have permissions to at least view the current
group, it could happen that the count returns 0. Make sure we return a
correct value even in that case.

balazssk committed:
LPS-164384 Language key

adolfopa committed:
LPS-166850 SF - Remove unused methods

# Conflicts:
#	modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContext.java

adolfopa committed:
LPS-166850 SF - Use interface instead of concrete class

balazssk committed:
LPS-164384 Wordsmith

balazssk committed:
LPS-164384 7.3 only has SegmentsExperienceConstants.ID_DEFAULT

balazssk committed:
LPS-164384 7.3 only has HttpUtil

balazssk committed:
LPS-164384 7.3 does not have PortletURLBuilder

balazssk committed:
LPS-164384 buildLang

brianchandotcom committed:
LPS-165197 Sort by exception class name

balazssk committed:
LPS-164384 7.3 has no externalReferenceCode

balazssk committed:
LPS-164384 7.3 Adapt by exporting getPortletNamespace and using correct sub

balazssk committed:
LPS-164384 7.3 uses LocalizationUtil instead of Reference

balazssk committed:
LPS-164384 7.3 remove data prefix

balazssk committed:
LPS-164384 7.3 Use current mvc.command.name

balazssk committed:
LPS-164384 Partial of "LPS-130145 Create infoItemPreviewSelectorURL" to include infoItems in page previews

balazssk committed:
LPS-164384 Partial of LPS-159329, but full commit "LPS-159329 Get selPlid from request parameters"

balazssk committed:
LPS-164384 Allow passing the version of the item and fill out the segmentsExperience in case we haven't any

# Conflicts:
#	modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/GetPagePreviewMVCResourceCommand.java

balazssk committed:
LPS-138258 Adds infoItemDetails attribute in preview

Commit:
LPS-167775 provide the portletID of where DDM is consumed

ces-quanhuynh committed:
LPS-164371 Add Integration Test

balazssk committed:
LPS-164384 New Language key not-selected

balazssk committed:
LPS-164384 buildLang

balazssk committed:
LPS-164384 7.3 Does not have updateAutoTags

balazssk committed:
LPS-164384 7.3 Adapt LPS-104110 based on UpdateArticleMVCActionCommand

markgulacsy committed:
LPS-139647 Change autoBatch usage

brianchandotcom committed:
LPS-164371 SF

ces-totrinh committed:
LPS-171030 Elasticsearch7 adds field mapping for userGroupRoleIds

kenjiheigel committed:
LRCI-3179 In Jenkins use ssh in docker browser container

jeffwu0724 committed:
LPS-165622 upgrade jackson-databind to version 2.13.4.2 in source-formatter.properties

jeffwu0724 committed:
LPS-165622 upgrade dependencies for jackson-databind

jeffwu0724 committed:
LPS-165622 auto sf

kenjiheigel committed:
LRCI-3179 Rework selectFieldText

kenjiheigel committed:
LRCI-3179 Add macrodef to get docker container ip addresses

kenjiheigel committed:
LRCI-3179 Improve docker error handling

kenjiheigel committed:
LRCI-3179 Pass in docker browser ip address to Poshi tests

kenjiheigel committed:
LRCI-3179 Add port for keycloak

kenjiheigel committed:
LRCI-3179 Use type instead of uploadFile

markgulacsy committed:
LPS-171082 Only update emailAddressVerified to true if the user is newly created during LDAP import

kenjiheigel committed:
LRCI-3179 Remove robotType usage

vendeltoreki committed:
LPS-167777 SF

kenjiheigel committed:
LRCI-3179 Open elasticsearch to additional IP addresses

cgoncas committed:
LPS-169907 Use original query, since we will lose the parameters in the collections query.

cgoncas committed:
LPS-169907 Performance improvement: we should check only the segmentEntryIds of the AssetList, instead check all the segmentsEntryIds of the system.

cgoncas committed:
LPS-169907 Avoid using non static utils as statics utils.

ealonso committed:
LPS-171240 We should check if the language id is a valid locale

(cherry picked from commit 0bd1c0f324e259b9a55eecfebc64674b97a4bb2a)

ces-hongvo committed:
LPS-171452 Adding Layout repository to export process

georgel-pop-lr committed:
LPS-171609 Add more space on top of the fields group and place + and - buttons for repeatable fields on very top

balazssk committed:
LPS-164384 7.3 Create DDMFormInputValue in pre Data Engine scenarios

balazssk committed:
LPS-171455 Display Page Preview does not display changes

jeffwu0724 committed:
LPS-155688 build-lib-versions

jeffwu0724 committed:
LPS-155688 SF

Wa1ter-Shao committed:
LRQA-78515 Unignore testUpdateDefaultLocale test and ignore testRemoveSupportedLocale test

ling-alan-huang committed:
LPS-171787 Auto SF

ling-alan-huang committed:
LPS-171787 Auto SF

ling-alan-huang committed:
LPS-171787 Auto SF

brianchandotcom committed:
artifact:ignore commerce-notification-web 3.0.15 artifact properties

brianchandotcom committed:
artifact:ignore commerce-notification-web 3.0.15 prep next

brianchandotcom committed:
artifact:ignore commerce-order-content-web 3.0.31 change log

brianchandotcom committed:
artifact:ignore commerce-order-content-web 3.0.31 artifact properties

brianchandotcom committed:
artifact:ignore commerce-order-content-web 3.0.31 prep next

brianchandotcom committed:
artifact:ignore commerce-order-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore commerce-order-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore commerce-order-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-method-authorize-net 3.0.21 change log

brianchandotcom committed:
artifact:ignore commerce-payment-method-authorize-net 3.0.21 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-authorize-net 3.0.21 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-method-mercanet 3.0.24 change log

brianchandotcom committed:
artifact:ignore commerce-payment-method-mercanet 3.0.24 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-mercanet 3.0.24 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-method-money-order 3.0.13 change log

brianchandotcom committed:
artifact:ignore commerce-payment-method-money-order 3.0.13 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-money-order 3.0.13 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-web 3.0.13 change log

brianchandotcom committed:
artifact:ignore commerce-payment-web 3.0.13 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-web 3.0.13 artifact properties

brianchandotcom committed:
artifact:ignore commerce-pricing-web 3.0.39 change log

brianchandotcom committed:
artifact:ignore commerce-pricing-web 3.0.39 artifact properties

brianchandotcom committed:
artifact:ignore commerce-pricing-web 3.0.39 prep next

brianchandotcom committed:
artifact:ignore commerce-product-definitions-web 6.0.47 change log

brianchandotcom committed:
artifact:ignore commerce-product-definitions-web 6.0.47 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-definitions-web 6.0.47 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-simple 3.0.19 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-simple 3.0.19 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-simple 3.0.19 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipment-web 3.0.21 change log

brianchandotcom committed:
artifact:ignore commerce-shipment-web 3.0.21 prep next

brianchandotcom committed:
artifact:ignore commerce-shipment-web 3.0.21 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipping-web 5.0.12 change log

brianchandotcom committed:
artifact:ignore commerce-shipping-web 5.0.12 prep next

brianchandotcom committed:
artifact:ignore commerce-shipping-web 5.0.12 artifact properties

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 3.0.17 change log

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 3.0.17 prep next

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 3.0.17 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 4.0.13 change log

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 4.0.13 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 4.0.13 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-checkout-web 4.0.31 change log

brianchandotcom committed:
artifact:ignore commerce-checkout-web 4.0.31 artifact properties

brianchandotcom committed:
artifact:ignore commerce-checkout-web 4.0.31 prep next

brianchandotcom committed:
artifact:ignore commerce-inventory-web 3.0.17 change log

brianchandotcom committed:
artifact:ignore commerce-inventory-web 3.0.17 artifact properties

brianchandotcom committed:
artifact:ignore commerce-inventory-web 3.0.17 prep next

brianchandotcom committed:
artifact:ignore commerce-price-list-web 4.0.20 change log

brianchandotcom committed:
artifact:ignore commerce-price-list-web 4.0.20 artifact properties

brianchandotcom committed:
artifact:ignore commerce-price-list-web 4.0.20 prep next

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 3.0.19 change log

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 3.0.19 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 3.0.19 prep next

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 3.0.16 change log

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 3.0.16 prep next

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 3.0.16 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 3.0.23 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 3.0.23 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 3.0.23 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 3.0.20 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 3.0.20 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 3.0.20 artifact properties

brianchandotcom committed:
artifact:ignore commerce-subscription-web 3.0.24 change log

brianchandotcom committed:
artifact:ignore commerce-subscription-web 3.0.24 prep next

brianchandotcom committed:
artifact:ignore commerce-subscription-web 3.0.24 artifact properties

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 3.0.20 change log

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 3.0.20 prep next

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 3.0.20 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-application-admin-web 4.0.27 change log

brianchandotcom committed:
artifact:ignore commerce-application-admin-web 4.0.27 prep next

brianchandotcom committed:
artifact:ignore commerce-application-admin-web 4.0.27 artifact properties

brianchandotcom committed:
artifact:ignore commerce-bom-admin-web 4.0.41 change log

brianchandotcom committed:
artifact:ignore commerce-bom-admin-web 4.0.41 artifact properties

brianchandotcom committed:
artifact:ignore commerce-bom-admin-web 4.0.41 prep next

brianchandotcom committed:
artifact:ignore commerce-machine-learning-forecast-alert-web 4.0.13 change log

brianchandotcom committed:
artifact:ignore commerce-machine-learning-forecast-alert-web 4.0.13 prep next

brianchandotcom committed:
artifact:ignore commerce-machine-learning-forecast-alert-web 4.0.13 artifact properties

brianchandotcom committed:
artifact:ignore commerce-organization-web 4.0.35 change log

brianchandotcom committed:
artifact:ignore commerce-organization-web 4.0.35 artifact properties

brianchandotcom committed:
artifact:ignore commerce-organization-web 4.0.35 prep next

brianchandotcom committed:
artifact:ignore commerce-punchout-web 3.0.17 change log

brianchandotcom committed:
artifact:ignore commerce-punchout-web 3.0.17 prep next

brianchandotcom committed:
artifact:ignore commerce-punchout-web 3.0.17 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-elasticsearch-monitoring-web 1.0.25 change log

brianchandotcom committed:
artifact:ignore portal-search-elasticsearch-monitoring-web 1.0.25 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-elasticsearch-monitoring-web 1.0.25 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 4.0.50 change log

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 4.0.50 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 4.0.50 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-lang 2.0.73 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-lang 2.0.73 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-lang 2.0.73 artifact properties

brianchandotcom committed:
artifact:ignore saml-hook 7.3.10.29 change log

brianchandotcom committed:
artifact:ignore saml-hook 7.3.10.29 prep next

brianchandotcom committed:
artifact:ignore saml-hook 7.3.10.29 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-lang 1.0.16 change log

brianchandotcom committed:
artifact:ignore search-experiences-lang 1.0.16 prep next

brianchandotcom committed:
artifact:ignore search-experiences-lang 1.0.16 artifact properties

brianchandotcom committed:
build.gradle auto SF

liuxingye committed:
LRQA-78725 Add path for check rename radio

liuxingye committed:
LRQA-78725 Add macro for rename when restore

liuxingye committed:
LRQA-78725 Modify macro for add file to subfolder by JSON

liuxingye committed:
LRQA-78725 Add tests for recycle bin

liuxingye committed:
LRQA-78725 Add test for recycle bin web content

liuxingye committed:
LRQA-78725 Move tests to the suitable file

Songyuewen committed:
LRQA-78315 Add macro to get pseudo element CSS value

Songyuewen committed:
LRQA-78315 Add test view column selected icon

dacousalr committed:
LPS-164384 7.3 SF

michaelhashimoto committed:
LRCI-3217 Add operating.system.types as a poshi property

michaelhashimoto committed:
LRCI-3217 Add operating system types to all the environment poshi tests

brookedalton committed:
LPS-172074 Set http.host if browser docker image is set

joshchong committed:
LPS-172187 Unquarantine Synonyms integration tests from search and relevant suites

brianchandotcom committed:
LPS-171656 prep next

HarryC0204 committed:
LRQA-78961 Quarantine testcase from rss relevant tests

ling-alan-huang committed:
LPS-171941 SF, sort

ling-alan-huang committed:
LPS-171941 SF, sort

georgel-pop-lr committed:
LPS-172224 Add portletId to the log

michaelhashimoto committed:
LRCI-3217 Add operating.system.types to the environment.acceptance PQL

ling-alan-huang committed:
LPS-171941 SF, sort

joaovictoralvesmartins committed:
LPS-162554 add language keys

joaovictoralvesmartins committed:
LPS-162554 run build lang

liferay-translation committed:
LPS-77699 Update Translations

tinatian committed:
LPS-172224 SF

brianchandotcom committed:
LPS-172224 use :

linolaoj committed:
LPS-163496 auto SF

rachaelkoestartyo committed:
LRAC-12672 fix JSONException

qz1010 committed:
LPS-172539 SF

brianchandotcom committed:
LPS-170848 LPS-172200 prep next

brianchandotcom committed:
LPS-167775 SF

brianchandotcom committed:
LRCI-3397 prep next

brianchandotcom committed:
Regen versions

ling-alan-huang committed:
LPS-172539 Auto SF

ling-alan-huang committed:
LPS-172539 SF

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LRCI-3399 Prep next

liferay-translation committed:
LPS-77699 Update Translations

Commit:
COMMERCE-10598 Sanitize addresses for safe HTML viewing

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

elocamp commented:
ci:test:sf
